examples/05_glm_second_level/plot_oasis.py

```diff
@@ -20,11 +20,16 @@
 We use only 100 subjects from the OASIS dataset to limit the memory usage.
 
 Note that more power would be obtained from using a larger sample of subjects.
+
+..
+    Original authors:
+
+    - Bertrand Thirion, <bertrand.thirion@inria.fr>, July 2018
+    - Elvis Dhomatob, <elvis.dohmatob@inria.fr>, Apr. 2014
```

Comment:
Note that Elvis and Vrigile's inria addresses are no longer valid, we should remove them.

Reply:
done